core/src/main/java/me/lokka30/treasury/plugin/core/command/subcommand/economy/migrate/CurrenciesMigrator.java

```diff
@@ -138,25 +138,21 @@ public boolean isPrimary() {
             @Override
             @NotNull
             public CompletableFuture<Response<BigDecimal>> parse(
-                    @NotNull String formatted,
-                    @Nullable Locale locale
+                    @NotNull String formatted, @Nullable Locale locale
             ) {
                 return c.parse(formatted, locale);
             }
 
             @Override
             public @NotNull String format(
-                    @NotNull BigDecimal amount,
-                    @Nullable Locale locale
+                    @NotNull BigDecimal amount, @Nullable Locale locale
             ) {
                 return c.format(amount, locale);
             }
 
             @Override
             public @NotNull String format(
-                    @NotNull BigDecimal amount,
-                    @Nullable Locale locale,
-                    final int precision
+                    @NotNull BigDecimal amount, @Nullable Locale locale, final int precision
```

Comment:
These are also worse but I'll accept it

Reply:
again, checkstyle

Reply:
can't be bothered fighting checkstyle, will just let it have its way